### ISSUE TITLE
"Avoid serving legacy JavaScript to modern browsers" Lighthouse report

### ISSUE THREAD

**Issue by nickdunn** [verbb/formie#801]
**Description**
Ok, these are incredibly marginal gains, but Lighthouse flags Formie with an "Avoid serving legacy JavaScript to modern browsers" comment.

<img width="565" alt="image" src="https://user-images.githubusercontent.com/39560/159046111-a576976f-bc2d-45b6-89ec-358d461e07b8.png">

Depending on whether you're supporting IE11 or not, a good few of these could be removed to bring down the JS bundle size.

**Comment by engram-design** [comment]
Yeah, we're supporting IE11 (until its EOL, June 15, 2022) - at least for the moment, but it's something we'll be looking at refactoring soon. We'll be switching from Laravel Mix to either Webpack or Vite, which better handles serving legacy bundles and polyfills.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.1](https://github.com/verbb/formie/releases/tag/4.0.0-beta.1).
